dcos_launch/config.py

```diff
@@ -610,13 +610,13 @@ def set_key_helper(platform: str, terraform_config: dict):
         'allowed': ['aws', 'gcp', 'gce', 'azure']},
     'terraform_config': {
         'type': 'dict',
-        'required': True},
+        'default': dict()},
     'init_dir': {
         'type': 'string',
         'default_setter': lambda doc: 'terraform-init-' + str(uuid.uuid4())},
     'terraform_dcos_version': {
         'type': 'string',
-        'default': 'master'},
+        'default': '95981fd0b1c3064d9e8d44b21575c20df09d94a5'},
```

Comment:
@falseperfection  I decided to pin the version because Terraform-dcos is changing quickly and has a high risk of breaking dcos-launch. However, having this as a param here leaves us the possibility to run against a newer version of terraform-dcos

Reply:
This is a tough choice to make on either side. It would be marginally preferable I think to link to a tag instead of a SHA but the same underlying issues are there

Reply:
Yeah. What I want to suggest is that we put master or "v1" here, but use the SHA as-needed to keep builds stable. So as features are added to terraform-dcos or it becomes more stable we don't have to keep updating and re-releasing dcos-launch.

Reply:
Follow up from our discussion in standup: hold on this PR until we can get a response from the terraform-dcos maintainters about providing a tag or branch that will have a stable API for us to pin against.

Reply:
@falseperfection after chatting with Miguel, it's better to leave the default to master and even always pull the latest terraform version. Since dcos-launch is a CI tool, doing this will help test new changes constantly on terraform-dcos. He does say that we should pin it for every new release.